test/session.test.js

```diff
@@ -798,6 +799,56 @@ test('only save session when it changes', async t => {
   t.equal(setCookieHeader2, undefined)
 })
 
+test('when rolling is true, keep saving the session', async t => {
```

Comment:
If the session already exists, we need to save the session again when `rolling` is set, regardless of whether or not the session was modified.

This replaces my previous code in this PR, when I used `maxAge` as a crutch for this.